dcat/rdf/dcat3.ttl

```diff
@@ -1,132 +1,152 @@
-# imports: http://purl.org/dc/terms/
-# imports: http://www.w3.org/2004/02/skos/core
-# imports: http://www.w3.org/ns/prov-o#
-# prefix: dcat
-
+@prefix adms: <http://www.w3.org/ns/adms#> .
+@prefix bibo: <http://purl.org/ontology/bibo/> .
 @prefix dcat: <http://www.w3.org/ns/dcat#> .
 @prefix dcterms: <http://purl.org/dc/terms/> .
 @prefix dctype: <http://purl.org/dc/dcmitype/> .
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix org: <http://www.w3.org/ns/org#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix prov: <http://www.w3.org/ns/prov#> .
 @prefix pav: <http://purl.org/pav/> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix sdo: <http://schema.org/> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
 @prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+@prefix xhv: <http://www.w3.org/1999/xhtml/vocab#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 <http://www.w3.org/ns/dcat>
   a owl:Ontology ;
-  dcterms:contributor [
+  adms:versionNotes "Esta es una copia del vocabulario DCAT 3 disponible en https://www.w3.org/ns/dcat.ttl"@es ;
+  adms:versionNotes "This is an updated copy the DCAT 3 vocabulary, taken from https://www.w3.org/ns/dcat.ttl"@en ;
+  adms:versionNotes "Dette er en opdateret kopi af DCAT 3 som er tilgænglig på https://www.w3.org/ns/dcat.ttl"@da ;
+  adms:versionNotes "Questa è una copia aggiornata del vocabolario DCAT 3 disponibile in https://www.w3.org/ns/dcat.ttl"@it ;
+  adms:versionNotes "Toto je aktualizovaná kopie slovníku DCAT 3, převzatá z https://www.w3.org/ns/dcat.ttl"@cs ;
+  bibo:editor [
       a foaf:Person ;
-      sdo:affiliation [
-          foaf:homepage <https://csiro.au> ;
-          foaf:name "Commonwealth Scientific and Industrial Research Organisation" ;
-        ] ;
-      rdfs:seeAlso <https://orcid.org/0000-0002-3884-3420> ;
-      foaf:name "Simon J D Cox" ;
-      foaf:workInfoHomepage <http://people.csiro.au/Simon-Cox> ;
-    ] ;
-  dcterms:contributor [
-      sdo:affiliation <http://www.w3.org/data#W3C> ;
-      rdfs:seeAlso <http://philarcher.org/foaf.rdf#me> ;
-      foaf:homepage <http://www.w3.org/People/all#phila> ;
-      foaf:name "Phil Archer" ;
+      foaf:homepage <https://w3id.org/people/ralbertoni/> ;
+      foaf:name "Riccardo Albertoni" ;
+      rdfs:seeAlso <https://orcid.org/0000-0001-5648-2713> ;
     ] ;
-  dcterms:contributor [
-      sdo:affiliation [
-          foaf:homepage <http://ec.europa.eu/dgs/informatics/> ;
-          foaf:name "European Commission, DG DIGIT" ;
-        ] ;
-      foaf:name "Vassilios Peristeras" ;
+  bibo:editor [
+      a foaf:Person ;
+      foaf:name "David Browning" ;
     ] ;
-  dcterms:contributor [
-      sdo:affiliation [
-          foaf:homepage <http://okfn.org> ;
-          foaf:name "Open Knowledge Foundation" ;
+  bibo:editor [
+      a foaf:Person ;
```

Comment:
Not sure I merit an 'editor' credit on v3. 
I've only been a distant participant ...